themes/stanford_basic/templates/components/paragraph--stanford-spacer.html.twig

```diff
@@ -0,0 +1,58 @@
+{#
```

Comment:
A template shouldn't be required. and why would there be a background anyways?

Reply:
I pulled this in from SDSS, where a template was used.  Is there a better way to attach the class to a div without doing it in a template?  I removed the background, which I was using for debugging.

Reply:
a preprocess hook should be sufficient to add the class as part of the attributes array.
This is where the behavior plugins make it easy. they have a build in method to do exactly that.

Reply:
Ok, that's a different approach than what they did in SDSS.  I'll see if I can refactor.